docs/sources/flow/reference/components/loki.source.docker.md

```diff
@@ -131,6 +131,14 @@ fully qualified name) to store its _positions file_. The positions file
 stores the read offsets so that if there is a component or Agent restart,
 `loki.source.docker` can pick up tailing from the same spot.
 
+In case the targets argument contains multiple entries with the same container
+ID (for example as a result of `discovery.docker` picking up multiple exposed
+ports or networks) `loki.source.docker` will deduplicate them, and only keep
```

Comment:
```suggestion
ID (for example as a result of `discovery.docker` picking up multiple exposed
ports or networks), `loki.source.docker` will deduplicate them, and only keep
```

Generally, we should avoid using brackets in the text. I'm leaving this one in for now because removing it makes for a VERY long sentence (or forces a big rewrite).